pkg/pdatautil/hash.go

```diff
@@ -63,6 +63,26 @@ func MapHash(m pcommon.Map) [16]byte {
 	return hw.hashSum128()
 }
 
+// MapHashSelectedKeys return a hash for the provided map, using values of only provided keys.
+// Order of hash calculation is determined by the order of keys.
```

Comment:
I see `MapHash` sorts keys to make sure the hash value doesn't change based on ordering. Should we do that here to?

Reply:
I thought since keys slice should be static, we could avoid the unnecessary work for the sake of performance. But I don't have a strong opinion on this.

Reply:
Have you been able to test to see if the hash consistently stays the same for metrics coming through a running instance of the collector? I just want to try to ensure it's not changing unexpectedly and taking away from the value of setting the key here.

If you've tested it and it works as expected, it's fine with me to keep it this way. If it turns out to be an issue down the road it would be a pretty simple and low-impact enhancement. 👍

Reply:
Checked again just now - hash is calculated consistently based on the order of keys in the config.